### ISSUE TITLE
Optic disc  excavation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7635]
**Preferred term label:**
Optic disc  excavation  -- what is the relation to  [Increased cup-to-disc ratio HP:0012796](https://hpo.jax.org/app/browse/term/HP:0012796)  

**Synonyms**
Excavation of optic discs 

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
615722 

PMID: 24462372   3/6

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@sukram Should there be a term Optic disc excavation or is this synonymous with Increased cup-to-disc ratio ?

**Comment by Sukramg** [comment]
The cup-to-disc ratio describes the extent of optic nerve excavation. In clinical parlance, cup-to-disc up to 0.6 is normal, 1.0 means complete cup-to-disc with virtually no nerve fiber fringe. 
Usually, the ratio is estimated by the examiner at the slit lamp and varies greatly from examiner to examiner. More accurate is the optical coherence tomography, as it is used for example for the follow-up of glaucoma.
I consider a graduation of the cup from 0 to 1.2 (vase-shaped cup) in 0.1 steps as a useful addition.

**Comment by pnrobinson** [comment]
@Sukramg  -- we now have these terms
https://hpo.jax.org/app/browse/term/HP:0031973
I will add Optic disc excavation as a synonym -- please contact me if this in incorrect!
